tests/data/parsers/test_parser_classification_rule.py

```diff
@@ -211,7 +211,11 @@ def test_feedback_for_criteria_with_gaps_and_overlap(
                 [2, "<0", "<0", ">10"],
                 [3, "<0", ">=0", "0:10"],
             ],
-            """For conditions: (varA: <0, varB: <0). Gap for variable varC in range -inf:0.0\nFor conditions: (varA: <0, varB: >=0). Gap for variable varC in range -inf:0.0\nFor conditions: (varA: <0, varB: >=0). Gap for variable varC in range 10.0:inf\nGap for variable varA in range 0.0:inf""",
+            [
+                call(
+                    """For conditions: (varA: <0, varB: <0). Gap for variable varC in range -inf:0.0\nFor conditions: (varA: <0, varB: >=0). Gap for variable varC in range -inf:0.0\nFor conditions: (varA: <0, varB: >=0). Gap for variable varC in range 10.0:inf\nGap for variable varA in range 0.0:inf"""
```

Comment:
there is a period missing at the end of the sentence
-inf:0.0.\nFor
also, shouldn't there be some spaces between: -inf : 0.0.?

Reply:
good point ;)